src/attached_probe.cpp

```diff
@@ -527,7 +527,13 @@ void AttachedProbe::load_prog()
     if (bt_verbose) {
       std::cerr << std::endl << "Error log: " << std::endl << log_buf << std::endl;
       if (errno == ENOSPC) {
-        std::cerr << "Error: No space left on device, try increasing BPFTRACE_LOG_SIZE environment variable" << std::endl;
+        std::stringstream errmsg;
+          errmsg << "Error: Failed to load program, verification log buffer "
```

Comment:
Is there any reason to use a stringstream here? Just a regular std::string would seem simpler to me

Reply:
I didn't start out to use `std::stringstream`, but the compiler complained about trying to concatenate `const char*` and `char[X]` strings, and `std::stringstream` worked around that nicely.

Looks like others have also done so in these locations of this file:

https://github.com/iovisor/bpftrace/blob/8b2d219236e757590c5e7bc388e179f625c00a27/src/attached_probe.cpp#L297
https://github.com/iovisor/bpftrace/blob/8b2d219236e757590c5e7bc388e179f625c00a27/src/attached_probe.cpp#L312
https://github.com/iovisor/bpftrace/blob/8b2d219236e757590c5e7bc388e179f625c00a27/src/attached_probe.cpp#L319

Reply:
Something like this should work too
```
        std::string msg = "Error: Failed to load program, verification log buffer "
                "not big enough, try increasing the BPFTRACE_LOG_SIZE "
                "environment variable beyond the current value of "
                + std::to_string(123) + " bytes";
```
But using a stringstream doesn't hurt either

Reply:
Will definitely keep that in mind.  Thanks for the tip!